test.js

```diff
@@ -206,18 +240,13 @@ test('deserialized name, stack and message should not be enumerable, other props
 		syscall: 'syscall',
 		randomProperty: 'random',
 	};
-	const enumerableProps = Object.keys(enumerables);
 
 	const deserialized = deserializeError({...object, ...enumerables});
-	const deserializedEnumerableProps = Object.keys(deserialized);
 
-	for (const prop of nonEnumerableProps) {
-		t.false(deserializedEnumerableProps.includes(prop));
-	}
-
-	for (const prop of enumerableProps) {
-		t.true(deserializedEnumerableProps.includes(prop));
-	}
+	t.deepEqual(
+		Object.keys(enumerables),
+		Object.keys(deserialized),
+	);
```

Comment:
I also took the opportunity to rewrite this after AVA complained that "`true` was expected but `false` was received." Not very useful. However `deepEqual` is _equally bad_ at reporting _exactly_ what's wrong 😰 

<img width="236" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/161102820-4bf9659c-dd8a-4b7e-a824-e22fcc9b0e60.png">